examples/time-elapsed-example.py

```diff
@@ -0,0 +1,20 @@
+from pypresence import Presence
+import time
+
+"""
+You need to upload your image(s) here:
+https://discordapp.com/developers/applications/<APP ID>/rich-presence/assets
+"""
+
+client_id = "client_id"  # Enter your Application ID here.
+RPC = Presence(client_id=client_id)
+RPC.connect()
+
+
+# Make sure you are using the same name that you used when uploading the image
+start_time=time.time()
```

Comment:
Could you please put some comments in to explain the large image & start_time args a bit better?

They're documented but lots of people expect to just learn from examples, so well-commented examples are appreciated.

Reply:
Ok, I'll get to it!